holidays/countries/sierra_leone.py

```diff
@@ -0,0 +1,132 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see AUTHORS.md file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from gettext import gettext as tr
+
+from holidays.groups import (
+    ChristianHolidays,
+    InternationalHolidays,
+    IslamicHolidays,
+    StaticHolidays,
+)
+from holidays.observed_holiday_base import (
+    ObservedHolidayBase,
+    SAT_SUN_TO_NEXT_WORKDAY,
+)
+
+
+class SierraLeone(
+    ObservedHolidayBase,
+    ChristianHolidays,
+    InternationalHolidays,
+    IslamicHolidays,
+    StaticHolidays,
+):
+    """Sierra Leone holidays.
+
+    References:
+        * https://www.officeholidays.com/countries/sierra-leone
+        * https://en.wikipedia.org/wiki/Public_holidays_in_Sierra_Leone
+        * https://www.timeanddate.com/holidays/sierra-leone/
+    """
+
+    country = "SL"
+    default_language = "en_SL"
+    # %s (estimated).
+    estimated_label = tr("%s (estimated)")
+    # %s (observed).
+    observed_label = tr("%s (observed)")
+    # %s (observed, estimated).
+    observed_estimated_label = tr("%s (observed, estimated)")
+    supported_languages = ("en_SL", "en_US")
+    # Sierra Leone gained independence on April 27, 1961
+    start_year = 1961
+
+    def __init__(self, islamic_show_estimated: bool = False, *args, **kwargs):
+        """
+        Args:
+            islamic_show_estimated:
+                Whether to add "estimated" label to Islamic holidays name
+                if holiday date is estimated.
+        """
+        ChristianHolidays.__init__(self)
+        InternationalHolidays.__init__(self)
+        IslamicHolidays.__init__(self, show_estimated=islamic_show_estimated)
+        StaticHolidays.__init__(self, SierraLeoneStaticHolidays)
+        kwargs.setdefault("observed_rule", SAT_SUN_TO_NEXT_WORKDAY)
+        super().__init__(*args, **kwargs)
+
+    def _populate_public_holidays(self):
+        dts_observed = set()
+
+        # New Year's Day.
+        dts_observed.add(self._add_new_years_day(tr("New Year's Day")))
+
+        # Armed Forces Day.
+        if self._year >= 2002:
+            dts_observed.add(self._add_holiday_feb_18(tr("Armed Forces Day")))
```

Comment:
```suggestion
        if self._year >= 2002:
            # Armed Forces Day.
            dts_observed.add(self._add_holiday_feb_18(tr("Armed Forces Day")))
```
Localization comment should be on the line preceding the line with localized string.